dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -295,6 +295,53 @@ internal class RumViewScopeTest {
         assertThat(updatedContext.applicationId).isEqualTo(fakeParentContext.applicationId)
     }
 
+    @Test
+    fun `M update the context with the viewType W initializing`(forge: Forge) {
+        // Given
+        val fakeViewEventType = forge.aValueFrom(RumViewScope.RumViewType::class.java)
+
+        // When
+        testedScope = RumViewScope(
+            mockParentScope,
+            fakeKey,
+            fakeName,
+            fakeEventTime,
+            fakeAttributes,
+            mockDetector,
+            mockCpuVitalMonitor,
+            mockMemoryVitalMonitor,
+            mockFrameRateVitalMonitor,
+            mockTimeProvider,
+            mockRumEventSourceProvider,
+            mockBuildSdkVersionProvider,
+            mockViewUpdatePredicate,
+            type = fakeViewEventType
+        )
+
+        // Then
+        assertThat(GlobalRum.getRumContext().viewType).isEqualTo(fakeViewEventType)
+    }
+
+    @Test
+    fun `𝕄 update the context with viewType NONE W handleEvent(StopView)`(
```

Comment:
Can you add a test 
`"𝕄 not update the context with viewType NONE W handleEvent(StartView) {another view is}"` to ensure we don't mess up the context from the next view